lib/edoc/src/edoc_extract.erl

```diff
@@ -32,15 +32,10 @@
 %% %% @headerfile "edoc.hrl" (disabled until it can be made private)
 -include("edoc.hrl").
 
-%% @type filename() = file:filename().
-%% @type proplist() = proplists:property().
-%% @type syntaxTree() = erl_syntax:syntaxTree().
-
-%% @spec source(File::filename(), Env::edoc_env(), Options::proplist())
-%%             -> {ModuleName, edoc:edoc_module()}
-%%    ModuleName = atom()
-%%    proplist() = [term()]
-%%
+-type filename() :: file:filename().
+-type proplist() :: proplists:property().
+-type syntaxTree() :: erl_syntax:syntaxTree().
+
```

Comment:
I disagree that aliasing types just to remove the module part makes it more readable.
When I skim through documentation there are enough types to keep track of, adding to that aliases which I need to make sure actually represent (and only represent) the ones module-prefixed it becomes noisier.